CHANGES.rst

```diff
@@ -2,9 +2,14 @@
 ==================
 
 - Bundled liberfa version update to v1.7.1.
+- Now it is possible to build against system liberfa [gh-39].
 - Improved the setup machinery to ensure a proper configuration of the
   embedded liberfa [gh-53] (see also https://github.com/liberfa/erfa/pull/73).
 - Improve version testing [gh-53] (see issue gh-52 for rationale).
+- Reworked version management [gh-57].
+- Make pyerfa build reproducible [gh-46] (see issue gh-45 for rationale).
+- Improved docstring titles [gh-47].
+- Most of the CI has been moved to GutHub Actions.
```

Comment:
Not sure this belongs in the user-facing `CHANGES.rst`, but if we keep it: GutHub-> GitHub

Reply:
done